eo-maven-plugin/src/test/java/org/eolang/maven/testapi/FakeMaven.java

```diff
@@ -109,42 +109,29 @@ public <T extends AbstractMojo> FakeMaven execute(final Class<T> mojo) {
     }
 
     /**
-     * Builds and returns compilation results after all manipulations.
-     *
-     * @return Compilation result.
-     */
-    public CompilationResult result() {
-        return new CompilationResult(
-            FakeMaven.PROGRAM_ID,
-            new Home(this.targetPath()),
-            this.foreignPath()
-        );
-    }
-
-    /**
-     * Creates eo-foreign.* file. In the future it is going to be a method for `MavenWorkspace`
-     * interface.
+     * Path to compilation target directory.
+     * @return Path to target dir.
      */
-    private void withEoForeign() {
-        Catalogs.INSTANCE.make(this.foreignPath())
-            .add(FakeMaven.PROGRAM_ID)
-            .set(AssembleMojo.ATTR_SCOPE, "compile")
-            .set(AssembleMojo.ATTR_EO, this.workspace.absolute(this.prog));
+    public Path targetPath() {
+        return this.workspace.absolute(Paths.get("target"));
     }
 
     /**
      * Path to 'eo-foreign.csv' or 'eo-foreign.json' file after all changes.
      * @return Path to eo-foreign.* file.
      */
-    private Path foreignPath() {
+    public Path foreignPath() {
         return this.workspace.absolute(FakeMaven.FOREIGN_PATH);
     }
 
     /**
-     * Path to compilation target directory.
-     * @return Path to target dir.
+     * Creates eo-foreign.* file. In the future it is going to be a method for `MavenWorkspace`
```

Comment:
@volodya-lombrozo I believe it's better to create new issue for "future" plan. No need to have it in JavaDoc.

Reply:
@mximp Sure. Removed.